nix/stack-shell.nix

```diff
@@ -3,7 +3,7 @@
 { system ? builtins.currentSystem
 , config ? {}
 , walletPackages ? import ./.. { inherit config system; }
-, pkgs ? walletPackages.pkgs
+, pkgs ? walletPackages._lib.pkgs
```

Comment:
Can `iohk-nix` be improved here? Getting the pinned pkgs seems like a fairly standard thing to do, so there should be a "standard" way to do it.